### PR TITLE
implement section 418

### PR DESCRIPTION
**Pull request by lucasmillanm**
### What?
section 418

### CODE REVIEW

**Review by stevan04 — APPROVED** [review]
all good

**Review by Nilstrieb — APPROVED** [review]

File: fabbwled-backend/src/main/resources/sections/397-432.yaml [hunk 1]
```diff
@@ -50,6 +50,20 @@
           then:
             - turnTo: 250
 
+- number: 418
+  content:
+    - text: |
```
Comment (keykey7) [inline fabbwled-backend/src/main/resources/sections/397-432.yaml:55]: `|` keeps newlines (`>` would not), see https://yaml-multiline.info/
...don't think we need a newline after "be" mid-sentence for example :)

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

**Review by Nilstrieb — APPROVED** [review]

**Review by miolmi — APPROVED** [review]

**Review by MagischerWolf05 — APPROVED** [review]
greate work

**Pull request merged**